public/js/components/Frames.js

```diff
@@ -52,7 +52,9 @@ function Frames({ frames, selectedFrame, selectFrame }) {
 
 module.exports = connect(
   state => ({
-    frames: getFrames(state).map(frame => {
+    frames: getFrames(state)
+    .filter(frame => getSource(state, frame.location.sourceId))
```

Comment:
nit: indentation is little weird here. Might be nice to indent it a little more to connect it with the first line?